week-1/exercises/paths.js

```diff
@@ -15,4 +15,7 @@ const base = filePath.slice(lastSlashIndex + 1);
 console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
+const DirPart = filePath.slice(1, 45);
```

Comment:
This solution works great for the example, but think about how we could extend the solution to work for different file paths (where we don't already know the length)